velox/functions/prestosql/tests/BinaryFunctionsTest.cpp

```diff
@@ -40,6 +41,7 @@ std::string hexToDec(const std::string& str) {
 }
 
 class BinaryFunctionsTest : public FunctionBaseTest {};
+class EncoderUtilsTest : public FunctionBaseTest {};
```

Comment:
We prefer one test class per file. In any case, I don't see EncoderUtilsTest being used. Its better to move this to common/encode/tests

Reply:
removed the class, and made a new test file as per velox standard